core/task_medium_test.go

```diff
@@ -134,7 +126,13 @@ func koRoutine(sch schedule.Schedule,
 	te := &taskErrors{
 		errs: make([]serror.SnapError, 0),
 	}
-	te.errs = append(te.errs, serror.New(errors.New("Dummy error")))
+	te.errs = append(te.errs, serror.New(
+		errors.New("Dummy error."),
```

Comment:
I added the dot because all error messages have dots in the end, for example:
```
// ErrPluginIncompatibleWithScheduleType - The error message for when a streaming schedule type references a non streaming plugin or vice versa.
ErrPluginIncompatibleWithScheduleType = errors.New("Plugin is incompatible with the tasks schedule type.")
// ErrMultipleStreamingPlugins - The error message when a task with a streaming schedule refers to multiple streaming plugins.
ErrMultipleStreamingPlugins = errors.New("Multiple streaming plugins within the same task is not supported.")
```